pandas/tests/frame/test_npfuncs.py

```diff
@@ -26,3 +26,11 @@ def test_np_sqrt(self, float_frame):
         assert result.columns is float_frame.columns
 
         tm.assert_frame_equal(result, float_frame.apply(np.sqrt))
+
+    def test_np_ravel(self):
+        # GH#26247 np.ravel() fails on list of DataFrames with column names
+        x = np.zeros((10, 3))
+        df = [DataFrame(batch.reshape(1, 3), columns=["1", "2", "3"]) for batch in x]
+        result = np.ravel(df)
+        expected = np.ravel([DataFrame(batch.reshape(1, 3)) for batch in x])
```

Comment:
It would be better if we could construct `expected` without using `ravel` (since that is what we are testing here). Could you simplify the data such that expected is small enough to just hardcode?